python/graphstorm/gconstruct/remap_result.py

```diff
@@ -806,10 +806,12 @@ def main(args, gs_config_args):
             #     task_id0/
             #     task_id1/
             #     ...
-            # We need to exclude both emb_info.json and task_id directories,
-            # when we are collecting node types with node embeddings.
+            # We need to exclude both emb_info.json, relation2id_map.json
+            # rel_emb.pt and task_id directories, when we are collecting
+            # node types with node embeddings.
             emb_names = [e_name for e_name in emb_names \
-                if e_name not in task_emb_dirs + ["emb_info.json"]]
+                if e_name not in task_emb_dirs + \
+                    ["emb_info.json", "relation2id_map.json", "rel_emb.pt"]]
```

Comment:
Do we have a standardized file name  regex for embeddings we could use ?  

We need a method that includes the correct files and only those, without us needing to update the code.

Reply:
After offline discussion let's ensure the retrieved path is a directory. We can strengthen the guarantee that every directory is a node type name later